src/openforms/js/components/form/email.js

```diff
@@ -1,8 +1,26 @@
 import {Formio} from "formiojs";
 import DEFAULT_TABS, {ADVANCED, SENSITIVE_BASIC, REGISTRATION, VALIDATION} from "./edit/tabs";
 
+const FormioEmail = Formio.Components.components.email;
+
+class EmailField extends FormioEmail {
+    static schema(...extend) {
+        return FormioEmail.schema({
+            validateOn: 'blur'
+        }, ...extend);
+    }
+
+    static get builderInfo() {
+        return {
+          title: 'Email',
+          group: 'advanced',
+          icon: 'at',
+          documentation: '/userguide/#email',
+          weight: 10,
+          schema: EmailField.schema()
+        };
+    }
```

Comment:
Do we need this override? And if we do, is the title still translatable?

Reply:
It seems that without it, the `validateOn` remains in its default value of `change`. 
It should remain translatable, because in the template the string gets marked as translatable: https://github.com/formio/formio.js/blob/master/src/templates/bootstrap/builderSidebarGroup/form.ejs#L38